mp4parse_capi/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,65 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use mp4parse_capi::*;
+use std::io::Read;
+
+type CursorType<'a> = std::io::Cursor<&'a [u8]>;
+
+extern "C" fn vec_read(buf: *mut u8, size: usize, userdata: *mut std::os::raw::c_void) -> isize {
+    let input: &mut CursorType = unsafe { &mut *(userdata as *mut _) };
+
+    let mut buf = unsafe { std::slice::from_raw_parts_mut(buf, size) };
+    match input.read(&mut buf) {
+        Ok(n) => n as isize,
```

Comment:
For the sake of soundness, I'd suggest
```suggestion
        Ok(n) => n.try_into().unwrap_or(-2),
```
or maybe just an `expect`.